packages/flutter/test/cupertino/button_test.dart

```diff
@@ -885,6 +885,66 @@ void main() {
     await tester.pump();
     expect(value, isTrue);
   });
+
+  testWidgets('Press and move on button and animation works', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      boilerplate(child: CupertinoButton(onPressed: () {}, child: const Text('Tap me'))),
+    );
+    final TestGesture gesture = await tester.startGesture(
+      tester.getTopLeft(find.byType(CupertinoButton)),
+      kind: switch (defaultTargetPlatform) {
```

Comment:
Nit: It's possible to have mouse events on iOS and Android, and it's possible to have touch events on Windows at least. That probably doesn't really matter here though since there's not distinction... I think this is fine as-is.